manifests/manifest.yaml

```diff
@@ -1,5 +1,14 @@
+  tls.key: LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlFcEFJQkFBS0NBUUVBdUN2cXBtWCs1WE9ncFMvT1hPOTZ4VUdOTTRtVURZMWszd0FCb09ZOE5wWm9rL0FHClBSTzRWNHpwaGtjZVdJWHVJc1FRV1RFelFLalFtQi9QMUF1TllvTDJUSWRiRUJMMW1IQWdEUzFEZ3A4MjB5c3gKd2FGS3YzWHRUSjg2VTFWVndYSW1QeXFNODhTVjI3VDNGRWYvb2NEckYxb3IzQnpWdmhMdVRMTFp5ek80NXhrdApQNUVueU1HQUtkbG5vU3F1V0dqa2lGSHh0bitITHpxbkEyYTBCeWQ0di9PdWpwdmdhelRiR00vYXdBbkNENmJBCkRZQVNOdXJjaWlvejlGNFM1VUwrSWpIemllU05ZNWpSMVBXNG9xNWEwS1F6ZjBybHdHT013cUltZlJXNFh4NDYKV21Sd2FJYmRPRDIvOFl5cEoxNGZ2Yk84R2duZVNmaCtMSUpRTlFJREFRQUJBb0lCQUYwaDhDNG5RK2ZhT2ZGdQpwR2VBdnMvRFgxa2hpRm4yRjMxaGJuRmtIQ3pxa0lYSENoQ1d5VkNVdEg2dnRodkQxbkdFUGxRc09pUEhlbnlECkVjREQrc0pIUWEvZmZ1QTVCak1JK0Z4UmVHTTBpaS90RnVNNDRpakJ4TGRMaWQ0Y09CcHowLzE3VWYzdmVteTUKTFh6M0Q4RkhZbHJUd1h3MHBLM1N1dWpVUktpcGFtL0oxSnNGeDVFYzZscndzZkFxc2FFOGJ5eXpmMXEzbzhMUApwc3doZ2xYS3RPdk5NK09UcXFyV3EveS94RXNHNFlETXl2L3FVL1RKaDV6V0I3cDdwbWpiSjdQMERBNXFwaTZ4CkEyditTNG1tQUNBSXRtSVp1U1c3aURPb01na1B4MEx4OWlVZ2M3S2RmTktPVW5kNTBTNU94VzJPakhYVXN2dFgKdHdmODhNRUNnWUVBMG5wS0cyN2o1MFZkNGduOFlNd2QrWjdKWFhMdk1sNkZXSWc5MktOSDRxdVpFTXV1TFJpWQphbUNwaFFqaU1heWxuZmRKdHB5ZXVrbGtTRm5waENuTzZQYjVzc0p4WkhQV3JpbU9KTVdUa1UwL2N5MVFwUU9uCkt1K0pGYnlzNStWQjlJNSs2ZEY2KzhEOVlLZEUwSTIzUVNKT0c4QWNIU0svNHdVd1NoZkVBUVVDZ1lFQTRBRWMKWDhIWHRaeGYyQzcyNjU2Z0NjdmNRVXg0NEw0NFFORW5HK0t2dWlhVXNqTG13RE9meisrVFAxdEFweVZ0UElmTAp1VXNGQkRzeFIvT3Q2bHphR2NaYWlBYktUSzZoMXRmS2w1UDRXcFEyNFZtVXg1V2QwWXUwbTNsY0tVMWJKcVV0CmVCdHJWSXI2VnRUWTJ0UW5oMlZoYjBKWlRvVDAxY1A4SWdJQStYRUNnWUVBZy80eXVTOTZkczkyaDhseFh4YkMKZVlkTmluQUlkMUwxVnNiaDJoalRmTi9WOXNWMHJrMHRVQkRjWGxScTNYSUN6cVNFc3FOWG0wVnBRVUk1Z2l0aAoyWkVBbUV6K0pWdkx0cDdTeUZhckswVUJWRzZIazJST0VrdnpjUkZJY1lqQkt1ZXR5VjRZUFFjVEh1am02ODdqCnJGSWlINThEUW5aSFZjd3NNMXJpRFMwQ2dZRUExcS9aM09Dd2Q4dStoRGhHeVAzSGpud0d4Y0tnRzIzVUJqTXUKazJoSVdWVytyNmRmUnBoenMxdCt1WHNnMU5GbkIxdHk0a3pIS3czOEptU2lxM3FBUWJHOGRLQmxaT2IrRHlYVQpja0FNQit3RFNTbU5FVmdRNEpOUDdxNTZxTThON0ZUazVqY0pRQWJPOVVlYjZ1Ym9WN2pQa0UxN3dHaG5LelZoCndjby8ydEVDZ1lCSkZIQTBIMFJZOGJERjBlcVVNeGVyWnNBM1lOY0RZS2JlMFE0dmJEbENLa3A3SFJRU3Q0Ri8KbTJSNThHemNzWU85VXNFbHhZTnNDdEJNVFMySFF0KytVbjloUGxrVWtqbjQ1dGtzUWVleDR2cW1FZkN2dmZPZQpKVWdRTi8vQ2JjdDJ5QTVNeVJuZEdrcEJBd0UyeUR2NlA2Skczai91dHVmRW1pVlJiNm5CU3c9PQotLS0tLUVORCBSU0EgUFJJVkFURSBLRVktLS0tLQo=
```

Comment:
Isn't this file generated with the generate-certs.sh? Should we really include it, even if it's a one-off cert-key combo?

Reply:
indeed, I would like to have a one-liner to install the webhook. The generate script is an additional step and not so failure-tolerant.  The user can end up with different ca and client cert, which will be confusing. I've added an extra warning now.